src/Cardano/Prelude/HeapWords.hs

```diff
@@ -441,7 +441,7 @@ instance HeapWords ByteString where
 
 instance HeapWords BSS.ShortByteString where
   heapWords s = let (w,t) = divMod (BSS.length s) wordSize
```

Comment:
```suggestion
  -- [SBS][*ByteArray#] -----> [BA#][sz][payload (word-aligned)]
  heapWords s = let (w,t) = divMod (BSS.length s) wordSize
```

Reply:
Thanks for the input. Went ahead and added better documentation.